instructions/instructions_impl.cpp

```diff
@@ -518,11 +520,23 @@ Sh::Sh(vector<string> args) {
 }
 
 void Sw::exec(State &state) {
-  state.stack[state.registers[dst] + offset + 3] = (std::byte) (state.registers[src] & 0xFF);
-  state.stack[state.registers[dst] + offset + 2] = (std::byte) ((state.registers[src] >> 8) & 0xFF);
-  state.stack[state.registers[dst] + offset + 1] = (std::byte) ((state.registers[src] >> 16) & 0xFF);
-  state.stack[state.registers[dst] + offset] = (std::byte) ((state.registers[src] >> 24) & 0xFF);
-  // The SW, SH, and SB instructions store 32-bit, 16-bit, and 8-bit values from the low bits of register rs2 to memory.
+/*
+  n = 4321
+  stack s
+  s[0] = 1 < *10^0
+  s[1] = 2 < *10^1
+  s[2] = 3 < *10^2
+  s[3] = 4 < *10^3
+*/
```

Comment:
I thought for a long time that this could matter, but then I realized what the point was)) 
In general looks right, great job!